src/ecs/components/draw/text.ts

```diff
@@ -197,7 +197,7 @@ export function text(t: string, opt: TextCompOpt = {}): TextComp {
             return t;
         },
         textSize: opt.size ?? DEF_TEXT_SIZE,
-        font: opt.font,
+        font: opt.font!,
```

Comment:
In the past we had problems with those !, specify why using on each case (or serie of cases)